### PR TITLE
Fix receivers list, and add warning.

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: ci/management/commands/generate_reports.py [hunk 1]
```diff
@@ -708,3 +709,13 @@ def handle_optouts_by_date(
                 "Receivers": ', '.join(row['receivers']),
                 "Number of Receivers": len(row['receivers']),
             })
+
+        # Add a warning to the sheet, because we cannot guarantee the best
+        # guess results
+        sheet.add_row({
+            "Timestamp": (
+                "NOTE: This datais not guaranteed to be correct, as "
```
Comment (erikh360) [inline ci/management/commands/generate_reports.py]: Typo: details

**Review by erikh360 — APPROVED** [review]
Just the type, otherwise +1

File: ci/management/commands/generate_reports.py (hunk 1 above)
Comment (rudigiesler) [reply ci/management/commands/generate_reports.py]: Ah, that's supposed to be `data is`. Or is it supposed to be `data are`?

**Comment by rudigiesler** [comment]
@erikh360 Fixed. Can you take another look?

**Review by erikh360 — APPROVED** [review]

**Pull request merged**